Description: Removed install prefix under lib directory

 Removed the lib/ctk-x.x eprefix, put all libs in /usr/lib.
 Removed version number from include directory
 .
 ctk (0.2.0+git20120918.f25f19e-1) experimental; urgency=low
 .
   * Upstream update
Author: Marco Nolden <dev@manol.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- ctk-0.2.0+git20120918.f25f19e.orig/CMakeLists.txt
+++ ctk-0.2.0+git20120918.f25f19e/CMakeLists.txt
@@ -140,10 +140,10 @@ if(NOT CTK_INSTALL_BIN_DIR)
   set(CTK_INSTALL_BIN_DIR "bin")
 endif()
 if(NOT CTK_INSTALL_LIB_DIR)
-  set(CTK_INSTALL_LIB_DIR "lib/ctk-${CTK_MAJOR_VERSION}.${CTK_MINOR_VERSION}")
+  set(CTK_INSTALL_LIB_DIR "lib")
 endif()
 if(NOT CTK_INSTALL_INCLUDE_DIR)
-  set(CTK_INSTALL_INCLUDE_DIR "include/ctk-${CTK_MAJOR_VERSION}.${CTK_MINOR_VERSION}")
+  set(CTK_INSTALL_INCLUDE_DIR "include/ctk")
 endif()
 if(NOT CTK_INSTALL_DOC_DIR)
   set(CTK_INSTALL_DOC_DIR "doc")
